README.md

```diff
@@ -24,3 +24,24 @@ Apache header:
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
     limitations under the License.
+
+Running the project:
+
+To post a job, you need to use the gcloud jobs submit training command to post a job using the
+module located in trainer.fintune:
+
```

Comment:
Generally you'd like to setup the readme in a way it's copy & paste and it just works. So it's useful to add the pre-defined defaults for first-time users.

i.e.

FINETUNING_STEPS=6000
MODEL_SIZE=base
MODEL_NAME=default

Reply:
Test comment: I changed the command to include the default numbers

Reply:
Another test comment